sktime/forecasting/base/convertIO/_convertIO.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+"""
+Machine type converters for scitypes
+
+Exports
+-------
+convert : dictionary of functions, with signature as below
+        indexed by triples of types
+         1st element = convert from - type
+         2nd element = convert to - type
+         3rd element = considered as this scitype - string
+        elements are conversion functions of machine type (1st) -> 2nd
+
+convert_to : function/dispatch version of convert, with 1st elt inferred
+
+
+-----------------------------------------
+
+Function signature of
+convert[(from_type, to_type, as_scitype)]
+
+Parameters
+----------
+what : from_type - object to convert
+store : reference of storage for lossy conversions, default=None (no store)
+
+Returns
+-------
+converted_what : to_type - object what converted to to_type
+
+Raises
+------
+ValueError and TypeError, if requested conversion is not possible
+                            (depending on conversion logic)
+
+-----------------------------------------
+
+Function signature of convert_to
+
+Parameters
+----------
+what : object to convert (any type)
+to_type : type, the type to convert "what" to
+as_scitype : str - name of scitype the object "what" is considered as
+store : reference of storage for lossy conversions, default=None (no store)
+
+Returns
+-------
+converted_what : to_type - object what converted to to_type
+
+Raises
+------
+TypeError if no suitable key in convert exists
+    i.e., (type(what), to_type, as_scitype)
+
+
+copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""
+
+
+__author__ = ["fkiraly"]
+
+
+import numpy as np
+import pandas as pd
+
+
+# dictionary indexed by triples of types
+#  1st element = convert from - type
+#  2nd element = convert to - type
+#  3rd element = considered as this scitype - string
+# elements are conversion functions of machine type (1st) -> 2nd
+
+convert = dict()
+
+
+def convert_identity(what, store=None):
+
+    return what
+
+
+# assign identity function to type conversion to self
+for tp in [pd.Series, pd.DataFrame, np.array]:
+    convert[(tp, tp, "Series")] = convert_identity
+
+
+def convert_UvS_to_MvS_as_Series(what: pd.Series, store=None) -> pd.DataFrame:
```

Comment:
maybe this should be called `convert_Series_to_Pandas_as_Series`? Do we know for certain that pandas frames will be the only machine type for MvS?

Reply:
>Do we know for certain that pandas frames will be the only machine type for MvS?

no, already the panel data conversions module allows numpy, so this may be something to align later on. I did not want to complicate things too much.

>maybe this should be called

It's supposed to be private and not called by the user. The only public exports are `convert` and `convert_to`

Reply:
You either misunderstood my suggestion or are joking...and I can't tell which

Reply:
I'm not sure why this would be funny

Reply:
> > maybe this should be called `convert_Series_to_Pandas_as_Series`
> 
> It's supposed to be private and not called by the user. The only public exports are `convert` and `convert_to`

You misinterpreted how I used the word `called` - I meant it in the sense of `named` or `labelled`, whereas you interpreted it as a call to a function/method. I couldn't tell if you genuinely misunderstood or if you were trying to make a pun.

Reply:
>Do we know for certain that pandas frames will be the only machine type for MvS?

I really hope so, but if we add another one, we can simply call it different from MvS.

Reply:
You have given two opposite answers to the question, "Do we know for certain that pandas frames will be the only machine type for MvS?" But I don't think it requires any immediate changes to the code though

> no, already the panel data conversions module allows numpy, so this may be something to align later on. I did not want to complicate things too much.

> I really hope so, but if we add another one, we can simply call it different from MvS.